analytics_data_api/v0/models.py

```diff
@@ -112,6 +112,25 @@ class Meta(object):
     created = models.DateTimeField(auto_now_add=True)
 
 
+class ProblemResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
+    """ Original model for the count of a particular answer to a response to a problem in a course. """
+
+    class Meta(BaseProblemResponseAnswerDistribution.Meta):
+        managed = False
+
+    count = models.IntegerField()
+
+
+class ProblemFirstLastResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
+    """ Updated model for answer_distribution table including a count of first attempts at problems. """
```

Comment:
I believe the comment would be more accurate if it included the last/final counts as well.